acceptance-tests/acceptance_tests_suite_test.go

```diff
@@ -142,3 +169,9 @@ func checkTLSErr(err error, expectString string) {
 	Expect(errors.As(tlsErr, &opErr)).To(BeTrue())
 	Expect(opErr.Err.Error()).To(ContainSubstring(expectString))
 }
+
+func println(s string) {
+	for _, line := range strings.Split(s, "\n") {
+		fmt.Printf("node %d/%d: %s\n", ginkgoConfig.GinkgoConfig.ParallelNode, ginkgoConfig.GinkgoConfig.ParallelTotal, line)
+	}
+}
```

Comment:
This shadows the builtin function `println(args ...Type)`. What about just `log(s string)`?

Reply:
We can't use `log` because that is a package in the stdlib which is imported in eg acceptance-tests/remote.go

Reply:
The stdlib function is fmt.Println FYI so it's not shadowed unless you import `fmt` as a dot import (which we're not doing anywhere)

I've changed the function name to `print` to avoid confusion with `fmt.Println`

Reply:
I'm talking about the builtin function, not the stdlib. You can call this without any package prefix and you can shadow them by defining something else with the same name. But IMO that's confusing and should be avoided. See https://pkg.go.dev/builtin#println (print is also a builtin).

Reply:
Ah never even heard of that.

I've changed it to `writeLog`